README.md

```diff
@@ -71,6 +71,18 @@ You can use MicroBin:
 
 ...and many other things, why not get creative?
 
+## How to deploy app into Fly.io server
+
+### Step 1:
+Install the [Fly.io CLI](https://fly.io/docs/hands-on/install-flyctl/)
+
+### Step 2:
+```
+git clone https://github.com/manjularajamani/microbin.git
```

Comment:
Is your fork of MicroBin needed for this to work, or can users clone szabodanika/microbin?

Reply:
Hi @szabodanika 

User can fork or clone [Szabodanika/microbin](https://github.com/szabodanika/microbin.git)